pyproject.toml

```diff
@@ -0,0 +1,2 @@
+[tool.black]
+line-length = 89
```

Comment:
YES! But this duplicates the same information on line 3 of `setup.cfg` ... why are both needed?

Reply:
The one in `setup.cfg` only applies to flake8, this is for black. Apparently no-one has been able to agree on the one true place to store such settings or a way to share common settings.

Reply:
However I now realize that I am missing a bullet point in TOOLS.rst about this settings file. I will add it.